docs/Using Fleet/MDM-OS-updates.md

```diff
@@ -38,6 +42,11 @@ End users are encouraged to update macOS (via [Nudge](https://github.com/macadmi
 | End user can defer                   | ✅                      | ✅                      | ❌                    |
 | Nudge window is dismissible          | ✅                      | ✅                      | ❌                    |
 
+### iOS and iPadOS (17.0+)
+
+For iOS and iPadOS (17.0+) devices that use Automated Device Enrollment (ADE), if the device is below the specified
```

Comment:
```suggestion
For iOS and iPadOS devices that use Automated Device Enrollment (ADE), if the device is below the specified
```

Reply:
It's already specified in the title which version of iOS  and iPadOS supports this